src/test/java/pl/javastart/movieclub/domain/comment/CommentServiceTest.java

```diff
@@ -97,12 +99,61 @@ void itShouldThrowUserNotFoundException() {
     }
 
     @Test
-    void itShouldFindPagedComments() {
+    void itShouldThrowExceptionWhenCommentNotFound() {
         //given
+        long id = 1L;
+        given(commentRepository.findById(id)).willReturn(Optional.empty());
+        //when
+        //then
+        assertThatThrownBy(() -> underTest.findCommentById(id))
+                .isInstanceOf(CommentNotFoundException.class)
+                .hasMessageContaining(String.format("Comment with ID %s does not exist.", id));
+    }
+
+    @Test
+    void itShouldUpdateComment() throws CommentNotFoundException {
+        //given
+        Comment updatedComment = new Comment();
+        updatedComment.setCommentContent("Comment content");
+
+        long id = 1L;
+        Movie movie = new Movie();
+        User user = new User();
+        Comment commentToUpdate = new Comment();
```

Comment:
Moze zamiast powielac w kazdym tescie tworzenie Comment wydziel to do osobnej metody i uzywaj tam gdzie potrzebne?

Reply:
tak zrobię jak będę update'ował wszystkie testy :)